cellprofiler/modules/trackobjects.py

```diff
@@ -186,8 +173,9 @@
 .. |TO_image1| image:: {PROTIP_RECOMEND_ICON}
 .. |TO_image2| image:: {PROTIP_RECOMEND_ICON}
 """.format(**{
-                "PROTIP_RECOMEND_ICON": PROTIP_RECOMEND_ICON
-                })
+    "PROTIP_RECOMEND_ICON": PROTIP_RECOMEND_ICON,
+    "HELP_ON_SAVING_OBJECTS": _help.HELP_ON_SAVING_OBJECTS
```

Comment:
It looks like `{HELP_ON_SAVING_OBJECTS}` should be added in the "See also" section, right above "Measurements made by this module".